### PR TITLE
add cn.xfangfang.wiliwili

### PR DESCRIPTION
**Pull request by pugaizai**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:https://github.com/xfangfang/wiliwili/issues/252**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by madoka773** [comment]
This flatpak will be maintained by the author @xfangfang

**Comment by madoka773** [comment]
bot, build

**Comment by madoka773** [comment]
bot, build

**Comment by madoka773** [comment]
bot, build cn.xfangfang.wiliwili

**Comment by madoka773** [comment]
wait, need some jobs for aarch64

File: cn.xfangfang.wiliwili.yaml [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+            config-opts: 
+              - '--disable-static'
+              - '--enable-libwebpmux'
+              - '--enable-libwebpdemux'
+            cleanup: 
+              - /share
+              - /bin
+            sources:
+              - type: archive
+                url: http://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.3.2.tar.gz
```
[the remark this answers is not in the document]
Comment (madoka773) [reply cn.xfangfang.wiliwili.yaml]: now it use prebuilt ffmpeg-full instead

**Comment by madoka773** [comment]
bot, build cn.xfangfang.wiliwili

**Comment by madoka773** [comment]
bot, build cn.xfangfang.wiliwili

**Comment by madoka773** [comment]
bot, build cn.xfangfang.wiliwili

**Comment by madoka773** [comment]
@xfangfang  it seeams that build failed caused by upstream opencc updates

**Comment by xfangfang** [comment]
@madoka773 It's fixed now.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
build failed.

**Comment by xfangfang** [comment]
bot, build cn.xfangfang.wiliwili

**Comment by madoka773** [comment]
> build failed.

it is successed now.

**Comment by barthalion** [comment]
bot, build cn.xfangfang.wiliwili

**Comment by madoka773** [comment]
@barthalion hello, can you add the app's author @xfangfang to the [flathub repo](https://github.com/flathub/cn.xfangfang.wiliwili)? thanks.

**Comment by bbhtt** [comment]
Can you please open an issue https://github.com/flathub/flathub/issues?

**Comment by madoka773** [comment]
Okay

**Pull request closed without merging**